<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuitareHero_Client</class>
 <widget class="QWidget" name="GuitareHero_Client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GuitareHero_Client</string>
  </property>
  <widget class="QLabel" name="labelTitre">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>401</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Bookman</family>
     <pointsize>48</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Piano Tile</string>
   </property>
   <property name="textFormat">
    <enum>Qt::MarkdownText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>411</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLineEdit" name="lineEditJoueur1">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="palette">
         <palette>
          <active>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>85</red>
              <green>87</green>
              <blue>83</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>85</red>
              <green>87</green>
              <blue>83</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>85</red>
              <green>87</green>
              <blue>83</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>239</red>
              <green>239</green>
              <blue>239</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>239</red>
              <green>239</green>
              <blue>239</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>85</red>
              <green>87</green>
              <blue>83</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>85</red>
              <green>87</green>
              <blue>83</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>85</red>
              <green>87</green>
              <blue>83</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <italic>false</italic>
          <bold>false</bold>
          <stylestrategy>PreferAntialias</stylestrategy>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(85, 87, 83);
</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLineEdit" name="lineEditJoueur2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(85, 87, 83);

</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLineEdit" name="lineEditJoueur3">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(85, 87, 83);
</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLineEdit" name="lineEditJoueur4">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(85, 87, 83);
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>510</y>
     <width>661</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_9">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Pseudo :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditPseudo"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="labelAdresseServeur">
              <property name="text">
               <string>Adresse du serveur :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditAdresseServeur">
              <property name="text">
               <string>172.18.58.81</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="labelNumeroPort">
              <property name="text">
               <string>Numero de port :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBoxPortServeur">
              <property name="minimum">
               <number>1024</number>
              </property>
              <property name="maximum">
               <number>65535</number>
              </property>
              <property name="value">
               <number>8888</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButtonConnexion">
          <property name="text">
           <string>Connexion</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonQuitter">
          <property name="text">
           <string>Quitter</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonPret">
        <property name="text">
         <string>Pas PrÃªt</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonQuitter</sender>
   <signal>clicked()</signal>
   <receiver>GuitareHero_Client</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>629</x>
     <y>524</y>
    </hint>
    <hint type="destinationlabel">
     <x>653</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
